src/nxt_http_parse.h

```diff
@@ -61,12 +61,10 @@ struct nxt_http_request_parse_s {
 
     /* target with "/." */
     uint8_t                   complex_target;         /* 1 bit */
-#if 0
     /* target with "%" */
     uint8_t                   quoted_target;          /* 1 bit */
     /* target with " " */
     uint8_t                   space_in_target;        /* 1 bit */
-#endif
```

Comment:
Curious why we are only now enabling these two things?

Reply:
See my new comment, but it looks like `space_in_target` is unused. Will check.